bitnami/zookeeper/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: zookeeper
-version: 5.7.2
+version: 5.7.3
```

Comment:
As you are adding a new feature this should be a minor change

Reply:
fixed